lightly/models/utils.py

```diff
@@ -596,3 +597,72 @@ def repeat_interleave_batch(x, B, repeat):
         dim=0,
     )
     return x
+
+
+def _get_2d_sincos_pos_embed(embed_dim, grid_size, cls_token=False):
+    """
+    Made 2d sincos positial embeddings. Code inspired by [0].
+
+    - [0]: https://github.com/facebookresearch/ijepa
+    grid_size: int of the grid height and width
+    return:
+    pos_embed: [grid_size*grid_size, embed_dim] or [1+grid_size*grid_size, embed_dim] (w/ or w/o cls_token)
```

Comment:
```suggestion
    Returns 2D sin-cos embeddings. Code from [0].

    - [0]: https://github.com/facebookresearch/ijepa
    
    Args:
        embed_dim:
            Embedding dimension.
        grid_size:
            Grid height and width. Should usually be set to sqrt(sequence length).
        cls_token:
            If True, a positional embedding for the class token is prepended to the returned
            embeddings.
    
    Returns:
        Positional embeddings array with size (grid_size * grid_size, embed_dim) if cls_token is False. 
        If cls_token is True, a (1 + grid_size * grid_size, embed_dim) array is returned.
```

Reply:
Could you adapt the other comments as well?

Reply:
@guarin Hi! I will do it, thanks for review

Reply:
@guarin please take a look.